cpu/stm32l1/periph/i2c.c

```diff
@@ -467,99 +411,94 @@ int i2c_write_regs(i2c_t dev, uint8_t address, uint8_t reg, char *data, int leng
 void i2c_poweron(i2c_t dev)
 {
     switch (dev) {
-#if I2C_0_EN
+
         case I2C_0:
             I2C_0_CLKEN();
             break;
-#endif
-#if I2C_1_EN
+
         case I2C_1:
             I2C_1_CLKEN();
             break;
-#endif
```

Comment:
@PeterKietzmann What do you mean with automatically? In which case should it used for?